### PR TITLE
Add some tests

### PR DESCRIPTION
**Pull request by xxhomey19**
Make code coverage over 90%

### CODE REVIEW

File: src/cli/shared/__tests__/getConfig.spec.js [hunk 1]
```diff
@@ -156,6 +156,25 @@ describe('Joi validate', () => {
     spy.mockRestore();
   });
 
+  it('should throw error if validate failed', () => {
+    const { MOCK_FILE_WITH_PLATFORM } = setup();
+    MOCK_FILE_WITH_PLATFORM.slack.accessToken = 12345;
```
Comment (kpman) [inline src/cli/shared/__tests__/getConfig.spec.js]: writing a comment will enhance readability, just like `// accessToken need to be string type`

File: src/cli/shared/__tests__/getConfig.spec.js [hunk 2]
```diff
@@ -156,6 +156,25 @@ describe('Joi validate', () => {
...
+
+    const configPath = 'bottender.config.js';
+    const platform = 'messenger';
+
+    const spy = jest.spyOn(Joi, 'validate');
+
+    expect(() => {
+      getConfig(configPath, platform);
+    }).toThrow();
+
```
Comment (kpman) [inline src/cli/shared/__tests__/getConfig.spec.js]: Add an assertion`expect(spy).toHaveBeenCalled();`? 
Or the spy function is useless.

File: src/context/__tests__/TelegramContext.spec.js [hunk 3]
```diff
@@ -101,6 +101,32 @@ it('get #client works', () => {
   expect(context.client).toBe(client);
 });
 
+describe('_getChatId', () => {
```
Comment (kpman) [inline src/context/__tests__/TelegramContext.spec.js]: Do we need to test private method?

Comment (xxhomey19) [reply src/context/__tests__/TelegramContext.spec.js]: not pretty sure, but it does be counted into testing coverage.

Comment (tw0517tw) [reply src/context/__tests__/TelegramContext.spec.js]: No, we shouldn't test private methods. We should test the public methods which use the private method you want to test.

**Pull request merged**